charmcraft/templates/init/CONTRIBUTING.md.j2

```diff
@@ -0,0 +1,16 @@
+# {{ name }}
+
+## Developing
```

Comment:
How about a additional heading with links to coding standards, best practice guidelines etc. ? Hopefully some day Juju SDK docs will have this ?

Reply:
I agree. I'll see if I can find some. For now, it's a little bit of a chicken-and-egg problem. If the Juju SDK docs don't currently have coding standards, best practice guidelines, etc, and we're actively discussing what they should be, it's hard to include them.

There can always be a follow-up patch which adds them once they're hashed out.